src/layouts/default.html.handlebars

```diff
@@ -36,11 +38,10 @@
                 <div class="fl-panelBar">
                     <span class="fl-prefsEditor-buttons">
                         <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
-                        <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                        <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide">+ Show Display Preferences</button>
```

Comment:
This change is not necessary: UIO will replace the button text. In fact, leaving it as "show/hide" is a helpful debugging aid, since it allows you to tell at a glance if UIO didn't initialize.